apstools/devices.py

```diff
@@ -1620,6 +1620,117 @@ def calibrate_energy(self, value):
         self.use_set.put("Use")
 
 
+class PVPositionerSoftDone(PVPositioner):
+    """
+    PVPositioner that uses an internal "done" signal.
+
+    Parameters
+    ----------
```

Comment:
This `---` line provokes a Sphinx `WARNING: Unexpected section title.`

See documentation in other devices for examples.